backend/src/resolvers/comments.spec.js

```diff
@@ -78,7 +78,37 @@ describe('CreateComment', () => {
         }
       }
 
-      await expect(client.request(createCommentMutation, createCommentVariables)).resolves.toMatchObject(expected)
+      await expect(
+        client.request(createCommentMutation, createCommentVariables)
+      ).resolves.toMatchObject(expected)
+    })
+
+    it('updates a comment', async () => {
+      await client.request(createCommentMutation, createCommentVariables)
+
+      const updateCommentMutation = `
+      mutation($postId: ID, $content: String!, $id: ID!) {
+        UpdateComment(postId: $postId, content: $content, id: $id) {
+          id
+          content
+        }
+      }
+      `
+      let updateCommentVariables = {
+        postId: 'p1',
+        content: 'Comment is updated',
+        id: 'c8'
```

Comment:
do you have a comment with the id `c8`... I can't see where you are creating that...